scripts/deploy/get-current-deployed-commit.sh

```diff
@@ -1,11 +1,7 @@
 #!/bin/bash
 
-BASE_URL=$REACT_APP_SPRUCE_URL
-if [ "$BASE_URL" == "" ]
-then
-    echo "BASE_URL not set"
-    exit 1
-fi
+BASE_URL=https://spruce.mongodb.com
```

Comment:
Ah, was an undefined `REACT_APP_SPRUCE_URL` the source of the problems on prod?

Reply:
yes the env variables wouldn't always be populated and it was random and difficult to reproduce. So I figured might as well remove the points of failure.